editor/scene_tree_dock.cpp

```diff
@@ -377,6 +377,9 @@ void SceneTreeDock::_tool_selected(int p_tool, bool p_confirm_override) {
 			if (!profile_allow_editing) {
 				break;
 			}
+			if (!_validate_no_foreign()) {
```

Comment:
I think the warning is unnecessary here if only single node is selected. The action will do nothing anyway.